snow/networking/sender/sender.go

```diff
@@ -1308,10 +1312,12 @@ func (s *sender) SendAppRequest(ctx context.Context, nodeIDs set.Set[ids.NodeID]
 
 			// Immediately register a failure. Do so asynchronously to avoid
 			// deadlock.
-			inMsg := message.InternalAppRequestFailed(
+			inMsg := message.InboundAppError(
```

Comment:
[no action required]. Not related to this changes but should we reduce indentation by using
```
if !s.timeouts.IsBenched(nodeID, s.ctx.ChainID) {
	continue
}
```
above?

Reply:
Leaving the above line untouched because it's not part of this pr